package.json

```
@@ -9,13 +9,13 @@
   "scripts": {
     "test": "test",
     "login": "clasp login",
-    "setup": "rm -f .clasp.json && clasp create --type sheets --title 'My React Project' --rootDir ./dist",
+    "setup": "rimraf .clasp.json && clasp create --type sheets --title \"My React Project\" --rootDir ./dist",
     "setup:use-id": "clasp setting rootDir dist && clasp setting scriptId",
-    "setup:https": "mkdir certs && mkcert -key-file ./certs/key.pem -cert-file ./certs/cert.pem localhost 127.0.0.1",
-    "build:dev": "NODE_ENV=development webpack",
-    "build": "NODE_ENV=production webpack",
-    "deploy:dev": "rm -rf dist && npm run build:dev && npx clasp push",
-    "deploy": "rm -rf dist && npm run build && npx clasp push",
+    "setup:https": "mkdirp certs && mkcert -key-file ./certs/key.pem -cert-file ./certs/cert.pem localhost 127.0.0.1",
+    "build:dev": "cross-env NODE_ENV=development webpack",
+    "build": "cross-env NODE_ENV=production webpack",
+    "deploy:dev": "rimraf dist && npm run build:dev && npx clasp push",
+    "deploy": "rimraf dist && npm run build && npx clasp push",
     "serve": "webpack-dev-server",
```

Do you mind adding the development environment for "serve"? Looks like I missed that one.

```suggestion
    "serve": "cross-env NODE_ENV=development webpack-dev-server",
```